### DISCUSSION TITLE
Welcome to vector-storage Discussions!

### DISCUSSION THREAD

**Discussion by nitaiaharoni1** [nitaiaharoni1/vector-storage#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stefanfmeyer** [comment]
Hey all!

Currently implementing this Vector Storage package into a Desktop app.

Fetching & Storing embeddings is going great, but I have an issue when trying to locate and embedding and update it in the store.

I am looking to find an embedding in the store based on it's value. 
In this case, I am storing the path for each embedding, as they are embeddings of full files.

I need to locate that embedding and fetch a new embedding, then update it.

Is there a function I'm unaware of that can locate the embedding? 
I've been struggling with this for a couple days, so thought I'd reach out in case I've missed something 😃
